### PR TITLE
(chore): remove the full restriction in archwsl.json

### LINKED ISSUE #7965: How about removing the 'full control access' restriction in archwsl.json
I have only tested it on win10 ltsc and it works without the full control access. For more details you can see https://github.com/yuk7/ArchWSL/issues/271#issue-1134979879

### PR DESCRIPTION
**Pull request by mxarcher**
Closes #7965

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/archwsl.json [hunk 1]
```diff
@@ -9,27 +9,18 @@
...
         "    warn 'Run ''Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-Linux'''",
         "    warn 'from an elevated PowerShell. Restart your computer when prompted and reinstall ArchWSL.'",
         "    $installable = $false",
         "}",
-        "$installable = $installable -and !(Invoke-ExternalCommand \"$dir\\Arch.exe\" 'isregd')",
+        "& \"$dir\\Arch.exe\" 'isregd'",
+        "if ($?) {",
+            "warn 'ArchWsl is already registered'",
+            "$installable = $false",
+        "}",
```
Comment (rashil2000) [inline bucket/archwsl.json:22]:
```suggestion
        "}",
```

File: bucket/archwsl.json [hunk 2]
```diff
@@ -58,3 +49,4 @@
         "url": "https://github.com/yuk7/ArchWSL/releases/download/$version/Arch.zip"
     }
 }
+
```
Comment (rashil2000) [inline bucket/archwsl.json:52]:
```suggestion
```

**Review by rashil2000 — CHANGES_REQUESTED** [review]

**Comment by rashil2000** [comment]
Can you also do this for alpinewsl and manjarowsl?

**Comment by mxarcher** [comment]
Thanks for your quickly reply, I've apply these changes to alpine and manjaro. However, I just test this on my win10 ltsc, I'm not sure whether it works on the others.

**Comment by rashil2000** [comment]
Thanks!

**Pull request merged**